va/va.h

```diff
@@ -2069,6 +2089,15 @@ typedef enum {
     */
     VAEncryptionParameterBufferType = 60,
 
+    /**
+     * \brief Encoding delta QP per block buffer
+     *
+     * This input buffer contains delta QP per block for encoding.
+     * Currently VAEncDeltaQpPerBlockBufferType is defined for HEVC encoding,
```

Comment:
better to add "the buffer only could be created and accepted when /c VAConfigAttribValEncPerMBControl delta_qp_support == 1".
please dont specify it in the common definitions, if you want to specify the bit length for each value, maybe could use some bits in VAConfigAttribValEncPerMBControl?

Reply:
Add the comments.

Reply:
Currently VAEncDeltaQpPerBlockBufferType is defined for HEVC encoding,see #VAEncDeltaQpPerBlockBuffer for details. ?? it was removed , right? ,maybe you should add detlaQP is a 16bit signed value for each block?

Reply:
An int8_t is enough for hevc.  Do we define it as an int16_t for other codec?